offchain-modules/packages/app-monitor/src/monitor.ts

```diff
@@ -140,6 +177,55 @@ export class Monitor {
     return false;
   }
 
+  async compareCkbLockAndEthMint(lock: CkbLockRecord, mint: EthMintRecord): Promise<boolean> {
+    const checker = (lock: CkbLockRecord, mint: EthMintRecord): string => {
+      if (lock.recipient.toLowerCase() !== mint.recipient.toLowerCase()) {
+        return `lock.recipient:${lock.recipient} !== mint.recipient:${mint.recipient}`;
+      }
+      return '';
+    };
```

Comment:
Add amount check，lockAmount should be gte mintAmount. Also add amount check to `compareEthLockAndCkbMint`

Reply:
added